Jenkinsfile

```diff
@@ -1,200 +1,4 @@
-                     env.DEPLOY_TO_PRD = input message: 'User input required',
-                                     submitter: 'lubneuskia,heatha',
-                                     parameters: [choice(name: 'dataservice-api: Deploy to PRD Environment', choices: 'no\nyes', description: 'Choose "yes" if you want to deploy the PRD server')]
-             }
-     }
-    }
-    stage('Retag with release'){
-      when {
-        environment name: 'DEPLOY_TO_PRD', value: 'yes'
-        expression {
-            return env.BRANCH_NAME == 'master';
-        }
-        expression {
-          return tag != '';
-        }
-      }
-      steps {
-        slackSend (color: '#005e99', message: ":deploying_qa: Retagging image with 'release'")
-        sh '''
-          MANIFEST=$(aws ecr batch-get-image --region us-east-1 --repository-name kf-api-dataservice --image-ids imageTag="prerelease-$tag" --query images[].imageManifest --output text)
-          aws ecr put-image --region us-east-1 --repository-name kf-api-dataservice --image-tag "$tag" --image-manifest "$MANIFEST"
-        '''
-      }
-    }
-    stage('Deploy PRD') {
-      when {
-       environment name: 'DEPLOY_TO_PRD', value: 'yes'
-       expression {
-           return env.BRANCH_NAME == 'master';
-       }
-       expression {
-         return tag != '';
-       }
-     }
-     steps {
-       slackSend (color: '#005e99', message: ":deploying_prd: DEPLOYING TO PRD: (${env.BUILD_URL})")
-       sh '''
-       kf-api-dataservice-config/ci-scripts/deploy_stage/deploy.sh prd
-       '''
-       slackSend (color: '#41aa58', message: ":white_check_mark: DEPLOYED TO PRD: (${env.BUILD_URL})")
-     }
-    }
-    stage("Rollback dataservice-api ?") {
-      when {
-             expression {
-               return env.BRANCH_NAME == 'master';
-             }
-             expression {
-               return tag != '';
-             }
-           }
-      steps {
-             script {
-                     env.ROLL_BACK = input message: 'User input required',
-                                     submitter: 'lubneuskia,heatha',
-                                     parameters: [choice(name: 'dataservice-api: Deploy to PRD Environment', choices: 'no\nyes', description: 'Choose "yes" if you want to deploy the PRD server')]
-             }
-     }
-    }
-    stage('Rollback PRD') {
-      when {
-       environment name: 'ROLL_BACK', value: 'yes'
-       expression {
-           return env.BRANCH_NAME == 'master';
-       }
-       expression {
-         return tag != '';
-       }
-     }
-     steps {
-       slackSend (color: '#005e99', message: ":deploying_prd: DEPLOYING TO PRD: (${env.BUILD_URL})")
-       sh '''
-       kf-api-dataservice-config/ci-scripts/rollback/rollback.sh prd
-       '''
-       slackSend (color: '#41aa58', message: ":white_check_mark: DEPLOYED TO PRD: (${env.BUILD_URL})")
-     }
-    }
-  }
+@Library(value='kids-first/aws-jenkins-shared-libraries') _
```

Comment:
Nice slimming down! Maybe I'm missing something, but when I went to hunt this down in Github, looks like it's actually `aws-infra-jenkins-shared-libraries` ?

Reply:
Well, @Library refers to the internal Jenkins config. So it is not pointed to the github. However, you are correct, maybe I should rename it to call aws-infra.... to stay consistent.